### PR TITLE
Don't launch if `--collect-only` is passed

### LINKED ISSUE #31: An empty launch reported to ReportPortal when i run 'pytest ---collect-only'
I think this is not what we expect.

My version:
python-reportportal   1.0.0
reportportal-client     3.0.0

### PR DESCRIPTION
**Pull request by s0undt3ch**
Fixes #31

### CODE REVIEW

File: pytest_reportportal/plugin.py [hunk 1]
```diff
@@ -8,26 +8,28 @@
...
-        tags=session.config.getini('rp_launch_tags'),
-        description=session.config.getini('rp_launch_description'),
-    )
-
-
-def pytest_sessionfinish():
-    # FixMe: currently method of RP api takes the string parameter
-    # so it is hardcoded
-    PyTestService.finish_launch(status='RP_Launch')
+    if session.config.getoption('--collect-only', default=False) is False:
```
Comment (krasoffski) [inline pytest_reportportal/plugin.py]: Here is my thoughts. 
I am OK with this change, but personally I prefer do not increase indentation level. 
Here is example how to do this in such way: 
```python
def pytest_sessionstart(session):
    if session.config.getoption('--collect-only', default=False):
         return
    PyTestService.init_service(
        project=session.config.getini('rp_project'),
        endpoint=session.config.getini('rp_endpoint'),
        uuid=session.config.getini('rp_uuid'),
        log_batch_size=int(session.config.getini('rp_log_batch_size')),
        ignore_errors=bool(session.config.getini('rp_ignore_errors')),
        ignored_tags=session.config.getini('rp_ignore_tags'))
```

File: pytest_reportportal/plugin.py [hunk 2]
```diff
@@ -8,26 +8,28 @@
...
+
+        PyTestService.start_launch(
+            session.config.option.rp_launch,
+            tags=session.config.getini('rp_launch_tags'),
+            description=session.config.getini('rp_launch_description'),
+        )
+
+
+def pytest_sessionfinish(session):
+    if session.config.getoption('--collect-only', default=False) is False:
```
Comment (krasoffski) [inline pytest_reportportal/plugin.py]: The same as above.

Comment (s0undt3ch) [reply pytest_reportportal/plugin.py]: Sure. Fixing.

**Comment by krasoffski** [comment]
@s0undt3ch If you do not agree with such approach I am OK with it and will merge the commit.

**Comment by s0undt3ch** [comment]
@krasoffski I share the same identation preference, I just hacked this too quickly. Anyway, fixed.

**Comment by krasoffski** [comment]
@ailjushkin I do not see any pitfalls for now. And if you agree with this changes, let's merge it.

**Comment by ailjushkin** [comment]
LGTM

**Pull request merged**